app/i18n/locales/terms-of-use/ada/zh-CN.md

```diff
@@ -0,0 +1,69 @@
+# 服务协议条款
```

Comment:
Example of file that didn't get its locale mapped.

Reply:
the changes probably need to be done in another PR to sync Crowdin